spec/rails_helper.rb

```diff
@@ -56,8 +56,8 @@ def with_okta_configured(&block)
     auth_server_metadata_url: 'https://example.com/oauth2/default/.well-known/oauth-authorization-server',
     issuer: 'https://example.com/oauth2/default',
     audience: 'api://default',
-    profile_api_url: 'https://example.com/api/v1/users/',
-    profile_api_token: 'token'
+    base_api_url: 'https://example.com/',
```

Comment:
This seems like it should be the same url as in  spec/lib/oidc/key_service_spec.rb

Reply:
Yes agreed, updating key_service now to accommodate.